modules/deploy/partials/cloud/iam-policies.adoc

```diff
@@ -744,3 +744,67 @@ Redpanda cluster resources:
 
 |===
 endif::[]
+
+ifdef::env-azure[]
+== Azure IAM policies
+
+IAM policies are assigned to deployed Redpanda agents for BYOC Azure
+clusters that use the following Azure services:
+
+  actions = [
+    "Microsoft.Resources/subscriptions/resourcegroups/read",
```

Comment:
These are just slightly out of date FYI: https://github.com/redpanda-data/cloudv2/blob/main/pkg/provisioners/azure/terraform/agent/iam.tf#L16

Reply:
> These are just slightly out of date FYI: https://github.com/redpanda-data/cloudv2/blob/main/pkg/provisioners/azure/terraform/agent/iam.tf#L16

Thanks @chapmanc, I've copied over the latest!